i2c/README.md

```diff
@@ -4,39 +4,35 @@ To get the embedded driver for your Sensirion Sensor working there are now
 only a few steps missing. These steps are needed since our driver is made
 in a way that it is possible to run it on different hardware platforms.
 So we need to adjust two files according to your platform.
-
-
-
 The first one is `sensirion_i2c_hal.c`.
-In this file it is implemented how to send your requests to the Sensor.
-How this is done depends on your hardware platform therefore we can't
+In this file it is implemented how to send your requests to the sensor.
```

Comment:
This file contains the implementation of the sensor communication (how to send requests to it)